pinot-common/src/main/java/org/apache/pinot/common/utils/helix/HelixHelper.java

```diff
@@ -55,6 +55,7 @@
   private static final String ENABLE_COMPRESSIONS_KEY = "enableCompression";
 
   private static final RetryPolicy DEFAULT_RETRY_POLICY = RetryPolicies.exponentialBackoffRetryPolicy(5, 1000L, 2.0f);
+  private static final RetryPolicy DEFAULT_TABLE_IDEALSTATES_UPDATE_RETRY_POLICY = RetryPolicies.fixedDelayRetryPolicy(20, 100L);
```

Comment:
I would use a random delay policy instead of fixed. Otherwise, you can have the same controllers coming back to update the IS at the same time.
I don't think we have the policy, so you can create one.

Reply:
added.